update with new guardian results (#10)
greenelab/nature_news_manuscript: README.md

<!-- usage note: edit this section. -->

> Science journalism is a critical way in which the public can remain informed and benefit from new scientific findings.

Reviewer: ```suggestion
Science journalism informs the public of the process and outcome of research as well as the implications of new scientific findings.
```

> Such journalism also shapes the public’s view of the current state of scientific findings and legitimizes experts.

Reviewer: ```suggestion
Such journalism also shapes the public’s view of scientists and legitimizes experts.
```

Those covering science can only cite and quote a limited number of sources.
Sources may be identified by the journalist’s research or by recommendations by other scientists.
In both cases, biases may influence who is identified and ultimately included as an expert.

> To ensure our results are generalizable, we analyzed two journalism outlets that cater to different demogrgaphics, _Nature_ and _The Guardian_.

Reviewer: ```suggestion
To ensure our results are generalizable, we analyzed two journalism outlets with different target audiences, _Nature_ and _The Guardian_.
```
